docs(telemetry): review privacy notice wording (#1403)
AmadeusITGroup/otter: docs/telemetry/README.md

# Telemetry Privacy Notice

> The Amadeus Otter team will put in place a new service: Telemetry; to collect some technical information and analyze the usage of the Otter builder and schematics provided in open-source.

Reviewer: ```suggestion
The Amadeus Otter team has put in place a telemetry service to collect and analyze technical information about how its open-source builders and schematics are used.
```
